Install failed on macOS 15.4 ARM

```
➜ brew install ymyzk/ymyzk/ochacaml
==> Auto-updating Homebrew...
Adjust how often this is run with HOMEBREW_AUTO_UPDATE_SECS or disable with
HOMEBREW_NO_AUTO_UPDATE. Hide these hints with HOMEBREW_NO_ENV_HINTS (see `man brew`).
==> Tapping ymyzk/ymyzk
Cloning into '/opt/homebrew/Library/Taps/ymyzk/homebrew-ymyzk'...
remote: Enumerating objects: 177, done.
remote: Counting objects: 100% (47/47), done.
remote: Compressing objects: 100% (26/26), done.
remote: Total 177 (delta 15), reused 43 (delta 13), pack-reused 130 (from 1)
Receiving objects: 100% (177/177), 29.75 KiB | 5.95 MiB/s, done.
Resolving deltas: 100% (82/82), done.
Tapped 1 command and 3 formulae (18 files, 46.8KB).
==> Fetching ymyzk/ymyzk/ochacaml
==> Downloading https://github.com/ymyzk/ochacaml/raw/140414/ochacaml.diff
==> Downloading from https://raw.githubusercontent.com/ymyzk/ochacaml/140414/ochacaml.d
################################################################################ 100.0%
==> Downloading https://caml.inria.fr/pub/distrib/caml-light-0.75/cl75unix.tar.gz
################################################################################ 100.0%
==> Installing ochacaml from ymyzk/ymyzk
==> Patching
==> Applying ochacaml.diff
/opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.3.8/lib/ruby/3.3.0/fileutils.rb:240: warning: conflicting chdir during another chdir block
==> make configure
Last 15 lines from /Users/chen/Library/Logs/Homebrew/ochacaml/01.make:
TERMINFO_DIRS=/opt/homebrew/Library/Homebrew/vendor/portable-ruby/current/share/terminfo
TMP=/private/tmp
TMPDIR=/private/tmp
TZ=UTC0
UID=501
USER=chen
VERSIONER_PERL_VERSION=5.34
ZERO_AR_DATE=1
_='Checking the sizes of integers and pointers...'
_JAVA_OPTIONS=-Duser.home=/Users/chen/Library/Caches/Homebrew/java_cache
__CF_USER_TEXT_ENCODING=0x1F5:0x0:0x0
cc='gcc -fno-defer-pop -D__FAVOR_BSD -no-cpp-precomp -lm'
Unable to compile the test program.
Make sure the C compiler is properly installed.
make: *** [configure] Error 2
```

--- a/Formula/ochacaml.rb
+++ b/Formula/ochacaml.rb
@@ -2,19 +2,12 @@ class Ochacaml < Formula
   desc "Shift/reset-extension of Caml Light"
   homepage "https://github.com/ymyzk/ochacaml"
   url "https://caml.inria.fr/pub/distrib/caml-light-0.75/cl75unix.tar.gz"
-  version "20140414"
+  version "20240719"
   sha256 "ac2ad4c7b28716ec3f0e2e24b8b092b99253a573ac8de7da396e08e8bcd10fd2"
 
-  bottle do
-    root_url "https://github.com/ymyzk/ochacaml/releases/download/140414"
-    rebuild 1
-    sha256 cellar: :any, sierra:     "9f6b344abbdab1db7619b1f73a18fe81ca23e10988ad9cee59ff694a0d567fd4"
-    sha256 cellar: :any, el_capitan: "c6dbc694c02a6d129a11e2fc893ed83eb34fb8e09c2615d07b72e65a0cc59d98"
-  end
-
   patch do
-    url "https://github.com/ymyzk/ochacaml/raw/140414/ochacaml.diff"
-    sha256 "c4ee52d2199ae7f7e9a4387b740e94653e9e458f6ca7da6f26fd6547fdfc48d9"
+    url "http://pllab.is.ocha.ac.jp/~asai/OchaCaml/download/OchaCaml.diff"
+    sha256 "f25735e8af920fc3cbd354670a57a396f5ebada062099244c55364b9ac568637"
   end
 
   def install

